app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    member_search = MemberSearch.new
+    member = member_search.find_member(user_params)
+    redirect_to "/address"
```

Comment:
```suggestion
    redirect_to "/find"
```
Since the new route to find an address is /find, this should be changed